sickbeard/webserve.py

```diff
@@ -1502,7 +1502,7 @@ def manualSearchCheckCache(self, show, season, episode, manual_search_type, **kw
         if kwargs:
             # Only need the first provided dict
             last_prov_updates = kwargs.iteritems().next()[0]
-            last_prov_updates = json.loads(last_prov_updates.replace("'", '"'))
+            last_prov_updates = json.loads(last_prov_updates.replace("'", '"')) if last_prov_updates else {}
         else:
```

Comment:
Fort want to know what the root cause is. Also, might to Reverence this issue in @Labrys webserve PR. Because the issue starts because the json arg is not processen correctly. So it's also a partly Tornado implementation issue.